### PR TITLE
[BISERVER-14864] - Empty file when downloading PIR to Excel 97-2003 format once data is past Excel limitations

### PR DESCRIPTION
**Pull request by renato-s**
Removed options to download .xls files

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 19m 54s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
designer/report-designer,designer/report-designer-extension-pentaho,engine/core
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>org.pentaho.reporting.ui.datasources.jdbc.ModuleTest.testModuleExists</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.mondrian.EmptyCellsetTest.testEmptyResult</b> (click to expand)</summary><p>

```
results should be empty, rowcount should be 0. expected:<0> but was:<1>
```

</p></details>

  
Tests run: **2279**, Failures: **2**, Skipped: **15** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1636/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 17m 40s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
designer/report-designer,designer/report-designer-extension-pentaho,engine/core
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>org.pentaho.reporting.ui.datasources.jdbc.ModuleTest.testModuleExists</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.mondrian.EmptyCellsetTest.testEmptyResult</b> (click to expand)</summary><p>

```
results should be empty, rowcount should be 0. expected:<0> but was:<1>
```

</p></details>

  
Tests run: **2279**, Failures: **2**, Skipped: **15** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1636/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: designer/report-designer-extension-pentaho/src/main/resources/org/pentaho/reporting/designer/extensions/pentaho/repository/messages_ja.properties [hunk 1]
```diff
@@ -37,8 +37,7 @@ RepositoryPublishDialog.ExportType.HTMLPage=HTML\u6587\u7ae0(\u30da\u30fc\u30b8\
 RepositoryPublishDialog.ExportType.CSV=\u6574\u5f62\u3055\u308c\u305fCSV\u6587\u7ae0
 RepositoryPublishDialog.ExportType.RTF=RTF\uff08\u30ea\u30c3\u30c1\u30c6\u30ad\u30b9\u30c8\u6587\u7ae0\uff09
 RepositoryPublishDialog.ExportType.TEXT=\u30c6\u30ad\u30b9\u30c8\u6587\u7ae0
-RepositoryPublishDialog.ExportType.XLS=Excel\u6587\u7ae0
-RepositoryPublishDialog.ExportType.XLSX=Excel2007\u6587\u7ae0
+RepositoryPublishDialog.ExportType.XLSX=Excel
```
Comment (smmribeiro) [inline designer/report-designer-extension-pentaho/src/main/resources/org/pentaho/reporting/designer/extensions/pentaho/repository/messages_ja.properties:40]: It should be "Excel2007\u6587\u7ae0", right?

File: designer/report-designer/src/main/resources/org/pentaho/reporting/designer/core/actions/messages/messages.properties [hunk 2]
```diff
@@ -644,7 +644,7 @@ PreviewExcelAction.Text=Excel
 PreviewXlsxAction.Accelerator=
 PreviewXlsxAction.Mnemonic=E
 PreviewXlsxAction.Description=Preview the Report as a Excel 2007 file
```
Comment (smmribeiro) [inline designer/report-designer/src/main/resources/org/pentaho/reporting/designer/core/actions/messages/messages.properties:646]: We can remove the '2007' from the description too - and, at the same time, please correct "a Excel" to "an Excel" :)

File: engine/core/src/main/resources/org/pentaho/reporting/engine/classic/core/modules/gui/xls/messages/messages.properties [hunk 3]
```diff
@@ -8,7 +8,7 @@ action.export-to-excel.accelerator=menu E
```
Comment (smmribeiro) [inline engine/core/src/main/resources/org/pentaho/reporting/engine/classic/core/modules/gui/xls/messages/messages.properties:8]: Can't we remove the 'action.export-to-excel.*' entries?

File: engine/core/src/main/resources/org/pentaho/reporting/engine/classic/core/modules/gui/xls/messages/messages.properties [hunk 4]
```diff
@@ -8,7 +8,7 @@ action.export-to-excel.accelerator=menu E
 
 action.export-to-xssf-excel.description=Save to MS-Excel 2007 format
 action.export-to-xssf-excel.mnemonic=VK_C
-action.export-to-xssf-excel.name=Export to Excel (2007) ...
+action.export-to-xssf-excel.name=Export to Excel ...
 action.export-to-xssf-excel.accelerator=menu shift E
 
 category.export.xls.name=Excel
```
Comment (smmribeiro) [inline engine/core/src/main/resources/org/pentaho/reporting/engine/classic/core/modules/gui/xls/messages/messages.properties:14]: Can't we remove the 'category.export.xls.*' entries?

**Review by smmribeiro — CHANGES_REQUESTED** [review]
Added a few comments.

**Review by smmribeiro — APPROVED** [review]
Looks good.

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 18m 40s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
designer/report-designer,designer/report-designer-extension-pentaho,engine/core
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>org.pentaho.reporting.ui.datasources.jdbc.ModuleTest.testModuleExists</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.mondrian.EmptyCellsetTest.testEmptyResult</b> (click to expand)</summary><p>

```
results should be empty, rowcount should be 0. expected:<0> but was:<1>
```

</p></details>

  
Tests run: **2279**, Failures: **2**, Skipped: **15** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1636/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**